helm/idol-qms/values.schema.json

```diff
@@ -109,6 +83,13 @@
                 },
                 "host": {
                     "type": "string"
+                },
+                "proxyBodySize":{
+                    "type":"string",
+                    "pattern": "^\\d+(?:\\.\\d+)?(?:Ki|Mi|Gi|Ti|Pi|Ei|m|k|M|G|T|P|E)?$"
+                },
+                "annotations": {
+                    "type": "object"
```

Comment:
Do these want sensible defaults in `values.yaml` and docs comments explaining what they are?

Reply:
Oh, looks like I added these to the idol-view/values.yaml but missed QMS/omnigroupserver.

Reply:
3784010